bases/shared/instagoric-server/server.js

```diff
@@ -548,30 +583,86 @@ privateapp.listen(privateport, () => {
   console.log(`privateapp listening on port ${privateport}`);
 });
 
-faucetapp.get('/', (req, res) => {
+
+faucetapp.get('/', async (req, res) => {
+
+  const denoms = await getDenoms();
+  let denomHtml = '';
+  denoms.forEach((denom) => {
+    denomHtml += `<label><input type="checkbox" name="denoms" value=${denom}> ${denom} </label>`;
+  })
```

Comment:
Are we sure there are no denoms we want to exclude? cc: @toliaqat

Reply:
These are testnets and I don't worry about that.